wandb/util.py

```diff
@@ -256,6 +256,10 @@ def get(key: str) -> Any:
             if value is not None and value != "":
                 scope.set_tag(tag, value)
 
+    # Track session so we can get metrics about error free rate
+    if sentry_hub:
+        sentry_hub.start_session()
```

Comment:
Could this be tested easily or do you need to dig deep into the sentry internals for that?
Also, there's no need to do "end_session" or something? :)

Reply:
We end up starting two sessions for each user process and the internal/service process.  This should be ok as we are mainly interested in crash free users.  The session is run by a timer, the default update rate is every 60 seconds so we are going to miss short sessions.   Let's see how this release goes and see if we need to do more to cover some of these caveats.